src/patternfly/components/Form/examples/Form.md

```diff
@@ -91,12 +91,25 @@ cssPrefix: pf-c-form
     {{#> form-label form-label--attribute=(concat 'for="' form--id '-simple-form-comment"')}}
       Comment
     {{/form-label}}
-    {{#> form-control controlType="input" input="true" form-control--modifier="pf-m-success" form-control--attribute=(concat 'value="This is a valid comment"' 'type="text" id="' form--id '-simple-form-comment" name="' form--id '-simple-form-comment" aria-describedby="' form--id '-simple-form-comment-help"')}}
+    {{#> form-control controlType="input" input="true" form-control--modifier="pf-m-success" form-control--attribute=(concat 'value="This is a valid comment" type="text" id="' form--id '-simple-form-comment" name="' form--id '-simple-form-comment" aria-describedby="' form--id '-simple-form-comment-help"')}}
     {{/form-control}}
     {{#> form-helper-text form-helper-text--modifier="pf-m-success" form-helper-text--attribute=(concat 'id="' form--id '-simple-form-comment-help" aria-live="polite"')}}
       This is helper text for success input
     {{/form-helper-text}}
   {{/form-group}}
+  {{#> form-group}}
+    {{#> form-label form-label--attribute=(concat 'for="' form--id '-simple-form-info"')}}
+      Information
+    {{/form-label}}
+    {{#> form-control controlType="textarea" form-control--attribute=(concat 'id="' form--id '-simple-form-info" name="' form--id '-simple-form-info" aria-invalid="true" aria-describedby="' form--id '-simple-form-info-helper"')}}
+    {{/form-control}}
+    {{#> form-helper-text form-helper-text--modifier="pf-m-error" form-helper-text--attribute=(concat 'id="' form--id '-simple-form-info-helper" aria-live="polite"')}}
```

Comment:
Could you update the handlebars file for `form-helper-text` so that if `HasIcon` is true, it adds the icon to the helper text.

Reply:
I don't think it always needs to be one kind of icon though, it can be anything. So you'd have to pass `form-helper-text-icon--type` or something, too, to define the icon type, but that gets tricky as you might want to pass something custom that isn't a `fas` icon. ^ is how we do this for most other components where you can pass a dynamic icon.

Reply:
Okay that makes sense